roles/nginx_uwsgi_config/files/nginx.conf

```diff
@@ -36,6 +36,9 @@ http {
     server_name _;
     server_tokens off;
 
+    # BB2-233 enabled CSP in report only mode
+    add_header Content-Security-Policy-Report-Only "default-src 'self' https://s3.amazonaws.com/content-dev-bbonfhir-com/ https://s3.amazonaws.com/content-test-bbonfhir-com/ https://s3.amazonaws.com/content-impl-bbonfhir-com/ https://s3.amazonaws.com/content-prod-bbonfhir-com/ https://ajax.googleapis.com https://stackpath.bootstrapcdn.com/bootstrap/ https://unpkg.com/feather-icons https://fonts.googleapis.com https://fonts.gstatic.com";
+
     # enforce HSTS policy (force clients to use HTTPS)
     add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload";
     add_header X-Content-Type-Options nosniff;
```

Comment:
just curious, still using X- headers?

Reply:
It is likely these were required at one point for compliance.